.travis.yml

```diff
@@ -1,12 +1,16 @@
 language: python
+services:
+  - redis-server
 python:
   - "3.5"
+before_install:
+  - "sudo apt-get install etcd -y"
 install:
   - "pip install tox"
 script:
-  - "tox -v -e py35"
+  - "etcd &"
```

Comment:
Highly nitpicky, but if you're still tweaking this you could `sudo systemctl start etcd` in `before_install:`.

Not a big deal though.